github4s/src/main/scala/github4s/interpreters/PullRequestsInterpreter.scala

```diff
@@ -160,4 +160,19 @@ class PullRequestsInterpreter[F[_]](implicit client: HttpClient[F]) extends Pull
       headers,
       reviewers
     )
+
+  override def updateBranch(
+      owner: String,
+      repo: String,
+      pullRequest: Int,
+      iAgreeToUseExperimentalApi: Boolean,
+      expectedHeadSha: Option[String] = None,
+      headers: Map[String, String] = Map()
+  ): F[GHResponse[BranchUpdateResponse]] =
+    client.put[BranchUpdateRequest, BranchUpdateResponse](
+      s"repos/$owner/$repo/pulls/$pullRequest/update-branch",
+      headers ++ Option("Accept" -> "application/vnd.github.lydian-preview+json")
```

Comment:
what happens if this header is not present?

Reply:
In this case an [unsupported media type is returned](https://docs.github.com/en/rest/reference/pulls#update-a-pull-request-branch): ```Status: 415 Unsupported Media Type```

Reply:
Then I think we need to remove the `iAgree...` and document it's experimental in the scaladoc and example, wdyt?

Reply:
Sure, that would be less confusing for users

Reply:
Would you look at the changes?